### ISSUE TITLE
Disabled actions should be read as "disabled"

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52577]
*Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : NVDA2018.1.1 / JAWS18.0.5038

**Repro Steps:**
1. Launch VS Code while NVDA/JAWS is ON.
2. Navigate to Activity Bar and select "Search"(Cntrl+Shift+F) button.
3. Navigate to disabled buttons above Search text field - "Refresh" , "Collapse All ", "Clear".

**Actual:**
Screenreader does not read the state of the disabled buttons above Search text field - "Refresh" , "Collapse All ", "Clear" as "disabled".

**Same issue has been observed with NVDA.**

**Expected:**
Screenreader should read the state of the disabled buttons above Search text field - "Refresh" , "Collapse All ", "Clear" as "disabled".

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screenreader users will not know about the state of the button and its availability for use.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[A11y_VSCode_ViewSearch_ScreenReader_Disabled.pptx](https://github.com/Microsoft/vscode/files/2123072/A11y_VSCode_ViewSearch_ScreenReader_Disabled.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
This is a general workbench issue, renaming. Assigning to july to investigate
fyi @bpasero

**Comment by bpasero** [comment]
I agree, ideally there is an `ARIA` role we could set for disabled buttons etc.

**Comment by jrieken** [comment]
verified tho it now skips those bottons
